src/crimson/mon/MonClient.cc

```diff
@@ -468,6 +471,27 @@ void Client::tick()
   });
 }
 
+seastar::future<> Client::wait_for_send_log() {
+  utime_t now = ceph_clock_now();
+  if (now > last_send_log + cct._conf->mon_client_log_interval) {
+    last_send_log = now;
+    return send_log(log_flushing_t::NO_FLUSH);
+  }
+  return seastar::now();
+}
+
+seastar::future<> Client::send_log(log_flushing_t flush_flag) {
+  bool flush = (flush_flag == log_flushing_t::FLUSH);
+  if (log_client) {
+    auto lm = log_client->get_mon_log_message(flush);
+    if (lm) {
```

Comment:
The `Client` is a quite modern thing that already uses the _if with initializer_ of C++17, so it would be desired to stick to the convention here.

Reply:
i would even suggest adding an additional interface to get_mon_log_client(), one that accepts
the enum. And in that case - move the enum def if needed.